python/hail/expr/__init__.py

```diff
@@ -0,0 +1,9 @@
+import hail.expr
+from hail.expr.functions import *
+from hail.expr.column import Column, NumericColumn
+from hail.expr.keytable import GroupedKeyTable, NewKeyTable
+from hail.expr.dataset import NewVariantDataset
+
+__all__ = ['NewKeyTable',
```

Comment:
Can you add the new classes to the docs?  Functions with no docstrings will just show up with function names/args, right?  Even that is probably useful for guinea pigs for the new interface.

Reply:
Question about this `__all__`.  Generally, I'd hope people could do `from hail2 import *` and they'd have everything they'd need to use the Hail interface, but that doesn't seem to be the case here because a ton of functionality comes through functions, but those aren't exported.  Is the intention to have functions in the default import or to do something like `import hail2.functions as hf` and then `hf.chisq(a, b, c, d)`.

There is good precedent for choosing a short (2-letter) import (e.g., np, tf, pd, ...) and accessing everything through that.  Something like:

```
import hail2 as hl
hc = hl.HailContext()
... hl.chisq(a, b, c, d) ...
```
What do you think?

Reply:
I was thinking it would be this:
```
from hail2 import *
import hail2.expr.functions as hf

hc = HailContext()
hf.chisq(a, b, c, d)
```

My concern with automatically importing the functions is they are more likely to conflict with other packages.  I think in this case it is better to be explicit.